### PR TITLE
strawberry: 1.1.3 -> 1.2.2

### PR DESCRIPTION
**Pull request by staticssleever668**
Closes #343933.
https://raw.githubusercontent.com/jonaski/strawberry/1.2.2/Changelog
Does this need a release note entry?

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by getchoo — CHANGES_REQUESTED** [review]
If this is being formatted, it should be done with [`nixfmt`](https://github.com/NixOS/nixfmt)

> Does this need a release note entry?

As you're dropping the Qt 5 version of the package, I would recommend it

File: nixos/doc/manual/release-notes/rl-2505.section.md [hunk 1]
```diff
@@ -27,6 +27,10 @@
 
 - `buildGoPackage` has been removed. Use `buildGoModule` instead. See the [Go section in the nixpkgs manual](https://nixos.org/manual/nixpkgs/unstable/#sec-language-go) for details.
 
+- strawberry has been updated to 1.2, which drops support for VLC backend and Qt 5.
+  The `strawberry-qt5` package has been removed. Use `strawberry-qt6` instead. Options `withGstreamer` and `withVlc`
+  have been removed, GStreamer is now the default.
```
Comment (getchoo) [inline nixos/doc/manual/release-notes/rl-2505.section.md:32]:
```suggestion
- `strawberry` has been updated to 1.2, which drops support for the VLC backend and Qt 5. The `strawberry-qt5` package and `withGstreamer`/`withVlc` override options have been removed due to this. 
```

File: pkgs/applications/audio/strawberry/default.nix [hunk 2]
```diff
@@ -60,52 +56,56 @@ stdenv.mkDerivation rec {
...
+      gnutls
+      kdsingleapplication
+      libXdmcp
+      libcdio
+      libebur128
+      libidn2
+      libmtp
+      libpthreadstubs
+      libtasn1
+      pcre
```
Comment (getchoo) [inline pkgs/applications/audio/strawberry/default.nix:74]:
```suggestion
      pcre2
```
We may be able to bump this -- should be in a separate commit in case we need to bisect, though. See also https://github.com/NixOS/nixpkgs/issues/356387

**Review by getchoo — CHANGES_REQUESTED** [review]

**Review by getchoo — APPROVED** [review]

**Comment by Scrumplex** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 358024`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 2 packages blacklisted:</summary>
  <ul>
    <li>nixos-install-tools</li>
    <li>tests.nixos-functions.nixos-test</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>strawberry (strawberry-qt6)</li>
  </ul>
</details>

**Review by Scrumplex — APPROVED** [review]
